dateparser/utils.py

```diff
@@ -99,25 +130,15 @@ def constructor(cls, *args, **kwargs):
             registry_dict = getattr(cls, "__registry_dict")
 
             if key not in registry_dict:
-                registry_dict[key] = creator(cls, *args, **kwargs)
+                registry_dict[key] = creator(cls, *args)
```

Comment:
this was necessitated by [this change in Python 3](https://hg.python.org/cpython/file/44ed0cd3dc6d/Objects/typeobject.c#l2818) for excessive parameters.